vignettes/tidying_casting.Rmd

```diff
@@ -88,7 +88,8 @@ Note that a tidier is also available for the `dfm` class from the quanteda packa
 library(methods)
 
 data("data_corpus_inaugural", package = "quanteda")
-d <- quanteda::dfm(data_corpus_inaugural, verbose = FALSE)
+d <- quanteda::tokens(data_corpus_inaugural) %>%
```

Comment:
I switched to `%>%` because the package as a whole still does. I'll open an issue about that.